src/main/java/org/jabref/gui/maintable/columns/LinkedIdentifierColumn.java

```diff
@@ -49,17 +50,31 @@ public LinkedIdentifierColumn(MainTableColumnModel model,
         this.setResizable(false);
         this.setCellValueFactory(cellData -> cellData.getValue().getLinkedIdentifiers());
         new ValueTableCellFactory<BibEntryTableViewModel, Map<Field, String>>()
-                .withGraphic(this::createIdentifierGraphic)
+                .withGraphic(values -> createIdentifierGraphic(values))
                 .withTooltip(this::createIdentifierTooltip)
                 .withMenu(this::createIdentifierMenu)
+                .withOnMouseClickedEvent((entry, linkedFiles) -> event -> {
+                    if ((event.getButton() == MouseButton.PRIMARY) && (linkedFiles.size() == 1)) {
+                        // Open linked identifier directly only if 1 entry is preset
+                        try {
+                            for (Field field : linkedFiles.keySet()) {
+                                JabRefDesktop.openExternalViewer(database, linkedFiles.get(field), field);
+                            }
+                        } catch (IOException e) {
+                            dialogService.showErrorDialogAndWait(Localization.lang("Unable to open link."), e);
+                        }
+                    }
+                })
                 .install(this);
     }
 
     private Node createIdentifierGraphic(Map<Field, String> values) {
-        if (values.isEmpty()) {
-            return null;
-        } else {
+        if (values.size() > 1) {
+            return cellFactory.getTableIcon(StandardField.URLS);
+        } else if (values.size() == 1) {
             return cellFactory.getTableIcon(StandardField.URL);
+        } else {
+            return null;
         }
```

Comment:
```suggestion
        if (values.size() > 1) {
            return IconTheme.JabRefIcons.MULTIPLE_LINKS.getGraphicNode();
        } else if (values.size() == 1) {
            return IconTheme.JabRefIcons.LINK.getGraphicNode();
        } else {
            return null;
        }
```
We talked in JabCon2020 a bit about this and we think that it would be better to directly load the icons here instead of introducing a new field `URLS` in `StandardFields` in the `model` package for a ui feature.

Reply:
@calixtus Yeah , that makes sense too , I have changed it to use JabRefIcons directly

Reply:
I could not find a multiple links icon , so for now I have left it as a such , with link and link-variant
some other alternatives are 
for single link
![Screenshot from 2020-09-01 14-53-30](https://user-images.githubusercontent.com/13389954/91832451-0c453100-ec63-11ea-8dbc-86a907d14d95.png)

for multiple 
![Screenshot from 2020-09-01 14-53-41](https://user-images.githubusercontent.com/13389954/91832403-00596f00-ec63-11ea-9c71-99c4c27d3af6.png)